setup.py

```diff
@@ -39,14 +39,14 @@
     long_description=long_description,
     keywords='dict, dictionary, mapping, bidirectional, bijection, bijective, injective, two-way, 2-way, double, inverse, reverse',
     url='https://github.com/jab/bidict',
-    license='ISCL',
+    license='ISC',
```

Comment:
I am only vaguely familiar with publishing packages on PyPI, so I can't comment on what value field should have on setup.py.

It is true that "ISCL" is prevalent, and I intend to work to reduce that. Right now, I am on a [mission to add the ISC license title to the license files](https://docs.google.com/spreadsheets/d/1YEDxL4Fp0tLGCCsRJjQDTpfMuGL-hnbhtUqMqwCEUco/edit#gid=1202575098), though, so that step will have to wait a bit :)

As for justification: as I mentioned in the other comment, the correct short identifier for the ISC license is simply "ISC". Given that [most of the licenses](https://pypi.python.org/pypi?%3Aaction=list_classifiers) that include an acronym do so because their full name doesn't include it, and many don't include an acronym at all, I supposed it would make sense to omit the redundant "(ISC)". That said, I have no strong objections to keeping it, if you prefer that -- but if so, the acronym should be "ISC", not "ISCL".

I realize this is not coherent with others like GPL, AFL, etc, but the license identifiers have worse incongruences than that :) In fact, if we were to insist on ISCL, then we should also advocate for MITL, BSDL, etc, which I don't believe would be practical.